client/src/main/java/com/kritsit/casetracker/client/domain/ui/controller/AdministratorController.java

```diff
@@ -126,7 +134,7 @@ private void editUser(){
             EditUserPane= (AnchorPane) fxmlLoader.load();
         }
         catch(IOException e){
-            logger.debug("Error loading frame to edit user " + e.toString());
+            logger.error("Error loading frame to edit user " + e.toString());
```

Comment:
Instead of logging the e.toString(), rather just pass the error as a parameter so the entire trace stack is logged